### PR TITLE
fix broken links

### PR DESCRIPTION
**Pull request by sumitshinde-84**
## Checklist

 - [ ] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

**Review by zackwasli — APPROVED** [review]

**Comment by sumitshinde-84** [comment]
Thanks @zackwasli !

File: src/handbook/customer/sales/subscription-agreement-1.4.njk [hunk 1]
```diff
@@ -125,7 +125,7 @@ PMB 24889, San Francisco, California 94104-5401 (or, if a different corporate en
 “User(s)” is defined as the unique and single Individual, employee, Contractor, or other third party individual or machine authorized by Customer (in accordance with this Agreement) that requires the provision of a seat within the admin platform, who are able to access the Software purchased under a Subscription, regardless of whether the User actually accesses or the frequency with which they access the Software. A User must be over the age of thirteen (13) years old.
 </p>
 <p>
-“Website” means FlowFuse’s website located at <a href="https://www.flowfuse.com">flowfuse.com</a>, <a href="https://www.flowforge.cloud">flowforge.cloud</a>,<a href="https://www.flowfuse.cloud">flowfuse.cloud</a> and all subdomains, and all content, services, documentation provided on the Website.
+“Website” means FlowFuse’s website located at <a href="https://www.flowfuse.com">flowfuse.com</a>, <a href="https://www.flowforge.com">flowforge.com</a>,flowfuse.cloud and all subdomains, and all content, services, documentation provided on the Website.
```
Comment (ZJvandeWeg) [inline src/handbook/customer/sales/subscription-agreement-1.4.njk:128]: @sumitshinde-84 @zackwasli why was this change merged? This is a legal definition, and should not be updated this way?

Comment (zackwasli) [reply src/handbook/customer/sales/subscription-agreement-1.4.njk:128]: It looked like these were simple link fixes, was trying to help while you were at the conference. Sorry if I merged something I shouldn't have!

Comment (ZJvandeWeg) [reply src/handbook/customer/sales/subscription-agreement-1.4.njk:128]: Not a problem, just making sure it's corrected

**Pull request merged**